java-support/src/main/scala/io/cloudstate/javasupport/impl/EntityDiscoveryImpl.scala

```diff
@@ -28,8 +28,8 @@ class EntityDiscoveryImpl(system: ActorSystem, services: Map[String, StatefulSer
   private val serviceInfo = ServiceInfo(
     serviceRuntime = sys.props.getOrElse("java.runtime.name", "") + " " + sys.props.getOrElse("java.runtime.version",
                                                                                               ""),
-    supportLibraryName = BuildInfo.name,
-    supportLibraryVersion = BuildInfo.version
+    supportLibraryName = sys.env.getOrElse("SUPPORT_LIBRARY_NAME", BuildInfo.name),
```

Comment:
Instead of pulling it from the env, I think it makes more sense to pull it from the system.settings.config

Reply:
Ok, i send commit later

Reply:
@sleipnir Awesome. Let me know if you need my input!

Reply:
Ok @viktorklang I sent a new attempt, can you check if you agree?